kim/mapper.py

```diff
@@ -5,12 +5,12 @@
 # This module is part of Kim and is released under
 # the MIT License: http://www.opensource.org/licenses/mit-license.php
 
-from six import with_metaclass
+import six
```

Comment:
Think we should probably import six like this throughout so it's obvious when you do something like `six.string_types` where it's coming from.